action-server/db/base.py

```diff
@@ -0,0 +1,35 @@
+import os
+
+from sqlalchemy import create_engine
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import sessionmaker
+
+TRACKER_STORE_USER = os.environ.get("SQL_TRACKER_STORE_USER")
+TRACKER_STORE_PASSWORD = os.environ.get("SQL_TRACKER_STORE_PASSWORD")
+TRACKER_STORE_URL = os.environ.get("SQL_TRACKER_STORE_URL")
+TRACKER_STORE_PORT = os.environ.get("SQL_TRACKER_STORE_PORT", "5432")
+TRACKER_STORE_DB = os.environ.get("SQL_TRACKER_STORE_DB")
+
+CONNECTION_POOL_SIZE = int(os.getenv("SQLALCHEMY_CONNECTION_POOL_SIZE", 15))
+CONNECTION_MAX_OVERFLOW = int(os.getenv("SQLALCHEMY_CONNECTION_MAX_OVERFLOW", 15))
+
+
+engine = create_engine(
+    "postgresql://{user}:{password}@{url}:{port}/{database_name}".format(
+        user=TRACKER_STORE_USER,
+        password=TRACKER_STORE_PASSWORD,
+        url=TRACKER_STORE_URL,
+        port=TRACKER_STORE_PORT,
+        database_name=TRACKER_STORE_DB,
+    ),
+    pool_size=CONNECTION_POOL_SIZE,
+    max_overflow=CONNECTION_MAX_OVERFLOW,
+)
+_SessionFactory = sessionmaker(bind=engine)
+
+
+Base = declarative_base()
```

Comment:
Can we have a more meaningful name that Base?

Reply:
usage of `Base` seems kind of widespread...